woocommerce-services.php

```diff
@@ -1748,6 +1764,17 @@ public function render_schema_notices() {
 				<?php
 			}
 		}
+
+		public function are_woo_shipping_and_woo_tax_active() {
+			$is_woo_shipping_active = in_array( 'woocommerce-shipping/woocommerce-shipping.php', get_option( 'active_plugins' ) );
+			$is_woo_tax_active      = in_array( 'woocommerce-tax/woocommerce-tax.php', get_option( 'active_plugins' ) );
+
+			return $is_woo_shipping_active && $is_woo_tax_active;
+		}
+
+		public function display_woo_shipping_and_woo_tax_are_active_notice() {
```

Comment:
[Non blocking] WP standards is normally pretty strict with adding DocBlocks, but it's not a strictly followed rule on the plugin anyway.